jujugui/static/gui/src/app/assets/css/_mixins.scss

```diff
@@ -203,3 +203,18 @@
     overflow-x: hidden;
     overflow-y: auto;
 }
+%shadow-header {
+    overflow: hidden;
+    padding-top: 0;
+    background: #fff;
+    color: $cool-grey;
+
+    header {
```

Comment:
switching to the new BEM workflows we don't target tags anymore because they influence too broadly.